python/swe/dockerfiles/create_index.py

```diff
@@ -11,6 +11,8 @@
 from composio.utils.logging import WithLogger
 
 
+print("This is a Test PR")
```

Comment:
The added print statement seems to be for debugging or demonstration purposes. Consider removing it before merging, or wrapping it in a conditional statement if it's intended for specific environments. If this print statement is meant to stay, consider adding a more informative message that explains its purpose or when it should be triggered.